packages/connectivity/src/scp-cf/destination/destination-cache.spec.ts

```diff
@@ -806,6 +807,41 @@ describe('destination cache', () => {
       expect(retrieveDestination()).toBeUndefined();
     });
   });
+
+  describe('custom destination cache', () => {
+    // Cache with expiration time
```

Comment:
consider also adding a test, where:
- default cache is used -> assert working
- switch to custom one -> assert working
- switch back to default -> still working old state there.

I have this in mind because potentially users want to use the global cache at a certain time or certain destinations and we must ensure that there are no side effects. Besides the obvious that each destination is just in one cache of course.

Reply:
As discussed, we would need to expose a function that returns the default cache instance which the user will maintain on their end and then switch to, by calling `setDestinationCache` again in case they want to use the default cache.
We can implement it later if this is requested by users.